prime-router/assets/csv-download-site/index__inline.html

```diff
@@ -18,10 +18,16 @@
             }
     
             function logout(){
-                removeItem("user")
-                removeItem("organization")
+                removeItem("jwt")
```

Comment:
Can you explain this change?

Reply:
that removes the cookies so back doenst work and we generally dont have the jwt token lying around.